charts/backingservices/README.md

```diff
@@ -70,48 +70,66 @@ Note: Only .p12 and .jks certificates are supported.
   deploymentName: "YOUR_SRS_DEPLOYMENT_NAME"
 
+  # Configure the location of the busybox image that is used during the deployment process of
+  # the internal Elasticsearch cluster
   busybox:
-    image: "busybox:1.31.0"
+    image: "alpine:3.16.0"
     imagePullPolicy: "IfNotPresent"
 
   srsRuntime:
-    #srs-service values
+    # Number of pods to provision
     replicaCount: 2
+
+    # docker image of the srs-service, platform-services/search-n-reporting-service:dockerTag
     srsImage: "YOUR_SRS_IMAGE:TAG"
-    imagePullPolicy: "IfNotPresent"
-    resources:
-      limits:
-        cpu: 1300m
-        memory: "2Gi"
-      requests:
-        cpu: 650m
-        memory: "2Gi"
-    serviceType: "ClusterIP"
+
     env:
-      #AuthEnabled may be set to true when there is an authentication mechanism in place between SRS and Pega Infinity.
+      # AuthEnabled may be set to true when there is an authentication mechanism in place between SRS and Pega Infinity.
       AuthEnabled: false
-      PublicKeyURL:
+      # Set the value for parameter 'PublicKeyURL' when AuthEnabled is true.
+      PublicKeyURL: ""
 
+  # This section specifies the elasticsearch cluster configuration.
   srsStorage:
-    # To configure authentication for the externally managed Elasticsearch cluster to use Basic Authentication then uncomment
-    # add the parameters, srs.srsStorage.basicAuthentication.username and srs.srsStorage.basicAuthentication.password
-    #    basicAuthentication:
-    #      username: "BASIC_AUTH_USERNAME"
-    #      password: "BASIC_AUTH_PASSWORD"
-    # To configure authentication for the externally managed Elasticsearch cluster to use an AWS IAM Role then uncomment and
-    # add the parameters, srs.srsStorage.awsIAM and srs.srsStorage.awsIAM.region
-    # srs.srsStorage.awsIAM's srs.srsStorage.awsIAM.region value
-    #    awsIAM:
-    #      region: "AWS_ELASTICSEARCH_REGION"
-
-    # set `requireInternetAccess` to true when the elasticsearch domain is outside of the Kubernetes cluster network  and is available over internet
-#    requireInternetAccess: true
+    # Setting srsStorage.provisionInternalESCluster to true will provision an internal elasticsearch cluster using the configuration
+    # specified in the `elasticsearch` section
+    provisionInternalESCluster: true
+    # To use your own Elasticsearch cluster, set srsStorage.provisionInternalESCluster to false and then
+    # set the external Elasticsearch cluster URL and port details below when using an externally managed elasticsearch
+    # Ensure that the specified endpoint is accessible from the kubernetes cluster pods.
+    #domain: ""
+    #port: 9200
+    #protocol: https
+    # The elasticsearch connection supports three authentication methods: basic authentication , AWS IAM role-based authentication and Elasticsearch secure connection(tls).
+    # To configure tls on internal cluster uncomment and set `srs.srsStorage.tls.enabled: true`
```

Comment:
To enable tls on Es Cluster uncomment and set `srs.srsStorage.tls.enabled: true`